.github/dependabot.yml

```diff
@@ -11,6 +11,6 @@ updates:
     schedule:
       interval: "daily"
   - package-ecosystem: "pip"
-    directory: "backend/"
+    directory: "backend/kesaseteli"
```

Comment:
why to limit on `kesaseteli` folder here? if there will be `backend/benefit` it does not get scanned.

Reply:
I fixed it @klemmari1